### ISSUE TITLE
About BN location

### ISSUE THREAD

**Issue by kaleidoscopical** [chenyilun95/tf-cpn#10]
Hi! Thank you for making this project open to the world! 
I run into some confuses because I am new to Tensorflow.
I wonder whether you add Batch Normalization(BN) to your added layers, e.g. 1*1 conv kernel and bottleneck in RefineNet.
Could you please give me a hint about where you add BN?

**Comment by chenyilun95** [comment]
That may be part due to historical problem. When we reproduce FPN for detection, we find BN can stablize the training process in case of  NaN loss, thus we inherit from that. In fact that incurs redundant overhead of computation if now it trains well. BN is applied in original bottleneck blocks in resnet. As far as I know, it can improve the performance.
